config/tf_modules/gcp-nodepool/main.tf

```diff
@@ -0,0 +1,54 @@
+resource "random_string" "node_pool_hash" {
+  length  = 4
+  special = false
+  lower   = true
+  upper   = false
+}
+
+resource "google_service_account" "gke_node" {
+  account_id   = "opta-${var.layer_name}-${random_string.node_pool_hash.result}"
+  display_name = "opta-${var.layer_name}-default-node-pool"
+}
+
+resource "google_container_node_pool" "node_pool" {
+  name               = "opta-${var.layer_name}-secondary"
```

Comment:
@juandiegopalomino @NitinAgg 
Should I rename it something else?

Reply:
module name

Reply:
Updated the name to be a layer_name + random_string